pkg/api/apiloader.go

```diff
@@ -41,6 +42,12 @@ func (a *Apiloader) DeserializeContainerService(contents []byte, validate bool)
 // LoadContainerService loads an ACS Cluster API Model, validates it, and returns the unversioned representation
 func (a *Apiloader) LoadContainerService(contents []byte, version string, validate bool) (*ContainerService, error) {
 	switch version {
+	case v20170831.APIVersion:
```

Comment:
Also, in the case of AgentPoolsOnly type, certificates will need to be passed as a separate params and not auto generated as is the case for ContainerService